documentation/kernel-semantics.tex

```diff
@@ -1576,30 +1556,35 @@ \subsubsection{Function Invocation}
             \cesktranswheresplit%
                 {\evalconf{\StaticGet{\membermeta}}{\env}{\strace}{\cstrace}{\cex}{\econt}}%
                 {\contconf{\econt}{\deref{\mainenv(\membermeta)}}}%
-                {\text{where $\membermeta$ is a static field and $\membermeta \in \dom(\mainenv$)}}
+                {\text{where $\membermeta$ is a static field and $\deref{\mainenv(\membermeta)} \neq \emptyset$}}
```

Comment:
Taking the note above into consideration, it should be `\deref{\mainenv(\membermeta)} \neq \inl()` here and further.